t/markdown.t

```diff
@@ -39,29 +46,4 @@ is(
   'Strikethrough extension'
 );
 
-my $table_markdown = <<'MARKDOWN';
-| Col1 | Col2 |
-| ---- |:----:|
-| val1 | val2 |
-MARKDOWN
-
-my $table_html = <<'HTML';
```

Comment:
Have absolutely no idea why this happens, but this table test seems to try to connect to the database?! Thus causing test_santiy to fail. Is there's some magic that causes a plain html string to make bmo want to connect to the database? Either way I feel like removing this test is not the end of the world.